app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'json'
+class ZomatoDataGrabbingWorker
+  include Sidekiq::Worker
+  include Faraday
+
+  def perform(*args)
+    puts 'Start Downloading Data From Zomato...'
+
+    conn = prepare_connection
```

Comment:
don't pass the connection between all those methods, it can be easily converted into a field